neqo-transport/benches/transfer.rs

```diff
@@ -27,45 +27,67 @@ const TRANSFER_AMOUNT: usize = 1 << 22; // 4Mbyte
+                        .pacing(pacing)
+                        .mlkem(false),
+                    boxed![ReachState::new(State::Confirmed)],
+                    boxed![ReceiveData::new(TRANSFER_AMOUNT)]
+                ),
+                TailDrop::dsl_downlink(),
+                RandomDelay::new(ZERO..JITTER),
+            ];
+            let mut sim = Simulator::new(label, nodes);
+            if let Some(seed) = &seed {
+                sim.seed_str(seed);
+            }
+            sim.setup()
+        };
+
+        let mut group = c.benchmark_group(format!("transfer/pacing-{pacing}/{label}"));
         group.noise_threshold(0.03);
-        group.bench_function(label, |b| {
+
+        // Benchmark with wallclock time, i.e. measure the compute efficiency.
+        group.bench_function("wallclock-time", |b| {
             b.iter_batched(
-                || {
-                    let nodes = boxed![
-                        Node::new_client(
-                            ConnectionParameters::default()
-                                .pmtud(true)
-                                .pacing(pacing)
-                                .mlkem(false),
-                            boxed![ReachState::new(State::Confirmed)],
-                            boxed![SendData::new(TRANSFER_AMOUNT)]
-                        ),
-                        TailDrop::dsl_uplink(),
-                        RandomDelay::new(ZERO..JITTER),
-                        Node::new_server(
-                            ConnectionParameters::default()
-                                .pmtud(true)
-                                .pacing(pacing)
-                                .mlkem(false),
-                            boxed![ReachState::new(State::Confirmed)],
-                            boxed![ReceiveData::new(TRANSFER_AMOUNT)]
-                        ),
-                        TailDrop::dsl_downlink(),
-                        RandomDelay::new(ZERO..JITTER),
-                    ];
-                    let mut sim = Simulator::new(label, nodes);
-                    if let Some(seed) = &seed {
-                        sim.seed_str(seed);
-                    }
-                    sim.setup()
+                setup,
+                |s| {
+                    black_box(s.run());
                 },
-                black_box(ReadySimulator::run),
                 SmallInput,
             );
         });
+
+        // Benchmark with simulated time, i.e. measure the network protocol
+        // efficiency.
+        //
+        // Note: Given that this is using simulated time, we can measure actual
+        // throughput.
+        group.throughput(criterion::Throughput::Bytes(TRANSFER_AMOUNT as u64));
+        group.bench_function("simulated-time", |b| {
+            b.iter_custom(|iters| {
+                let mut d_sum = Duration::ZERO;
+                for _i in 0..iters {
+                    // run() returns the simulated time, excluding setup time.
```

Comment:
```suggestion
                    // run() returns the simulated time, excluding setup time.
                    // No need for black_box(), because this doesn't measure compute.
```